.github/workflows/integration-tests-grpc.yml

```
@@ -15,8 +15,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { java-version: adopt@1.8,      scala-version: 2.13.3,  sbt-opts: '' }
-          - { java-version: adopt@1.11.0-9, scala-version: 2.13.3,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
+          - { java-version: adopt@1.8,      scala-version: 2.13.x,  sbt-opts: '' }
+          - { java-version: adopt@1.11, scala-version: 2.13.x,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
```

```suggestion
          - { java-version: adopt@1.8,  scala-version: 2.13.x,  sbt-opts: '' }
          - { java-version: adopt@1.11, scala-version: 2.13.x,  sbt-opts: '-J-XX:+UnlockExperimentalVMOptions -J-XX:+UseJVMCICompiler' }
```

Am I being a formatting freak? 
(feel free to ignore it 😄 )